operator/pkg/resources/configuration/configuration_test.go

```diff
@@ -32,23 +32,23 @@ func TestRedpandaProperties(t *testing.T) {
 	config := configuration.GlobalConfiguration{Mode: configuration.GlobalConfigurationModeCentralized, NodeConfiguration: rpkcfg.ProdDefault()}
 	config.SetAdditionalRedpandaProperty("a", "b")
 	require.NoError(t, config.FinalizeToTemplate())
-	concreteCfg, err := config.ConcreteConfiguration(context.TODO(), nil, "", nil)
+	concreteCfg, err := config.ConcreteConfiguration(context.TODO(), nil, nil, "", nil)
```

Comment:
NIT: This change should be applied to the next commit.

Reply:
I believe you guys squash anyway so should be fine :)